CHANGELOG.md

```diff
@@ -5,7 +5,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
-
+### Added
```

Comment:
Suggesting version number for this release.
 
```suggestion

## [1.11.3] - 2021-11-18
### Added
```

Reply:
changed